### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by ostcar** [OpenSlides/OpenSlides#6369]
### Discussed in https://github.com/OpenSlides/OpenSlides/discussions/6364

<div type='discussions-op-text'>

<sup>Originally posted by **111phil** February 17, 2022</sup>
We were contacted by participants with disabilities that it is quite hard for them to use OpenSlides, since it is not compatible with assistive technologies. This should be enhanced in the future. </div>
